src/modules/pact-consumer-core.module.ts

```diff
@@ -44,7 +44,7 @@ export class PactConsumerCoreModule {
       {
         provide: useClass,
         useClass,
-      },
+      } as ClassProvider,
```

Comment:
Unless I am missing something, I don't think this should be converted using `as` - I think it would be better to make sure that the passed object is the right type.